### PR TITLE
Test Content

### PR DESCRIPTION
**Pull request by asafshen**
fixes: https://github.com/demisto/server/issues/7857

@meirwah I will also create another issue with all tasks from today's meeting

also - this PR contains some test playbooks minor fixes

### CODE REVIEW

File: Integrations/integration-Cisco-umbrella.yml [hunk 1]
```diff
@@ -410,8 +410,8 @@ script:
         });
 
         context = {
-            IP: {
-                Address: address,
+            Domain: {
+                Name: domain,
```
Comment (meirwah) [inline Integrations/integration-Cisco-umbrella.yml]: these are bug fixes ? something else?

File: Tests/conf.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+    "testTimeout": 30,
```
Comment (meirwah) [inline Tests/conf.json]: is this a must? or there are defaults?

File: circle.yml [hunk 3]
```diff
@@ -14,5 +19,37 @@ test:
     - cp Reports/* bundle/
     - cp content-descriptor.json bundle/
     - cp $(find Scripts -type f -print) bundle/
-    - cd bundle/ && zip ../content.zip * 
+
+    - if [ "$CIRCLE_BRANCH" != "master" ]; then cp TestPlaybooks/* bundle/ ; fi
```
Comment (meirwah) [inline circle.yml]: isn't this duplicate logic as create_content_artifact.sh ?

File: Tests/scripts/create_instance.sh [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+#configure aws
+aws configure set region us-west-2
+
+IMAGE_ID="ami-d2c924b2" # centos
+
+#create instance
+INSTANCE_ID=$(aws ec2 run-instances \
+    --image-id ${IMAGE_ID} \
+    --security-group-ids sg-714f3816 \
+    --instance-type t2.micro \
```
Comment (meirwah) [inline Tests/scripts/create_instance.sh]: you might want to consider bigger vm for faster build

File: Tests/test_integration.py [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+    IN_PROGRESS = 'inprogress'
+
+
+# ----- Functions ----- #
+
+
+# get integration configuration
+def __get_integration_config(client, integration_name):
+    res = client.req('POST', '/settings/integration/search', {
+        'page': 0, 'size': 100, 'query': 'name:' + integration_name  # TODO - fix query
```
Comment (meirwah) [inline Tests/test_integration.py]: you have a TODO here

File: Tests/test_integration.py [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+        r = client.CreateIncident(name, None, None, None, None,
+                         None, None, **kwargs)
+    except RuntimeError as err:
+        print_error(str(err))
+
+    response_json = r.json()
+    inc_id = response_json['id']
+
+    # wait for incident to be created
+    time.sleep(1)
```
Comment (meirwah) [inline Tests/test_integration.py]: better to do polling here.. maybe 1 sec is to much...

File: circle.yml [hunk 7]
```diff
@@ -1,11 +1,16 @@
...
-    - ./yamlvalidator
-    - chmod +x ensurefilenames.sh
-    - ./ensurefilenames.sh
+  pre:
+    # install python client
+    - pip install git+https://github.com/demisto/demisto-py.git@test-content
+
+    # validate yaml
+    - chmod +x ./Tests/yamlvalidator
+    - ./Tests/yamlvalidator
```
Comment (meirwah) [inline circle.yml]: did you add somewhere to check that each integration/script has a description here?

**Comment by meirwah** [comment]
@asafshen plz verify build is stable before merging

File: TestPlaybooks/playbook-Cisco-Meraki-Test.yml [hunk 8]
```diff
@@ -1,5 +1,5 @@
-id: 4c8954af-af6f-4c67-82c4-3d1d21bd34b6
-version: 45
+id: Cisco-Meraki-Test
+version: 1
```
Comment (dorsha) [inline TestPlaybooks/playbook-Cisco-Meraki-Test.yml]: Version should be -1

File: TestPlaybooks/playbook-Qualys-Test.yml [hunk 9]
```diff
@@ -1,5 +1,5 @@
-id: dd897871-cc85-4d0f-80db-8877aedf193c
-version: 3
+id: Qualys-Test
+version: 1
```
Comment (dorsha) [inline TestPlaybooks/playbook-Qualys-Test.yml]: Also here

File: Tests/scripts/installer_commands.exp [hunk 10]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/expect -f
+
+set timeout -1
```
Comment (dorsha) [inline Tests/scripts/installer_commands.exp]: Need to use the -y option

File: Integrations/integration-Cisco-umbrella.yml (hunk 1 above)
Comment (asafshen) [inline Integrations/integration-Cisco-umbrella.yml]: yes its a fix

File: Tests/conf.json (hunk 2 above)
Comment (asafshen) [inline Tests/conf.json]: Added a default also

File: circle.yml (hunk 7 above)
Comment (asafshen) [inline circle.yml]: its on the roadmap :)

**Pull request merged**